script/Common.s.sol

```diff
@@ -349,6 +237,118 @@ abstract contract Common is Contracts, Params {
     }
   }
 
+  function _revokeAllTo(address _governor) internal {
+    if (!_shouldRevoke()) return;
```

Comment:
ohhh boy, well, you started it:
- i'm ok with the reorder
- i think `_revoke` could be named `_revokeDeployerTo` or sth more descriptive
- i don't like we're re-writing the loop twice in `_delegateAllTo`

i'm thinking the following, if it's possible:
```solidity
function toAllContracts(function(IAuthorizable) returns (bool) _function, address _target) {
    _function(safeEngine, _target);
    ...
}
```

wdyt about?